docs/components/InputText/Web.stories.tsx

```diff
@@ -57,7 +61,6 @@ Toolbar.args = {
 
 export const Readonly = BasicTemplate.bind({});
 Readonly.args = {
-  defaultValue: "Rocinante",
```

Comment:
What's going on here? I can now update text on a Readonly input. Left is prod, right is local

![Screenshot 2025-01-17 at 1 50 43 PM](https://github.com/user-attachments/assets/e0117a66-9f78-4c47-bba3-f77896a00747)

Reply:
I realized I messed up since version 2 uses `readOnly` and version 1 uses `readonly`. Another issue was I had pushed a change to the `BasicTemplate` that used the version 2 input for testing before I spent the time on making the version comparison story.

Reply:
The commit that fixed this can be found here https://github.com/GetJobber/atlantis/pull/2316/commits/64b4a8cf34a8dbc34717d9a5e06950768b2688bd

Reply:
Oh nice catch @taylorvnoj !!!